aws_lambda_builders/workflows/nodejs_npm/utils.py

```diff
@@ -53,3 +55,41 @@ def is_windows(self):
     def parse_json(self, path):
         with open(path) as json_file:
             return json.load(json_file)
+
+
+def get_install_action(source_dir, artifacts_dir, subprocess_npm, osutils, build_options):
```

Comment:
I wasn't really sure where to put this function since it's used in two separate workflows, open to suggestions.

Reply:
I'm unsure which one is better between here or in the nodejs_npm workflow

Reply:
Moved it to the workflow, seems to make more sense there.